src/components/Whiteboard/ElementBehaviors/Moveable/MoveableElement.tsx

```diff
@@ -46,11 +43,15 @@ const DraggableGroup = styled('g')({
 
 export type MoveableElementProps = PropsWithChildren<{
   elementId: string;
+  overrides?: Elements;
 }>;
 
-export function MoveableElement({ children, elementId }: MoveableElementProps) {
-  const { activeElementIds } = useActiveElements();
-  const elements = useElementOverrides(activeElementIds);
+export function MoveableElement({
+  children,
+  elementId,
+  overrides = {},
```

Comment:
This can become

```suggestion
  overrides: elements = {},
```

then the assignment `const elements = overrides;` can go away.

Reply:
this is what I went for.